src/lsm/manifest.zig

```diff
@@ -317,17 +319,25 @@ pub fn ManifestType(comptime Table: type, comptime Storage: type) type {
         }
 
         pub fn assert_no_invisible_tables(manifest: *const Manifest, snapshot: u64) void {
-            for (manifest.levels) |*manifest_level| {
-                var it = manifest_level.iterator(
-                    .invisible,
-                    @as(*const [1]u64, &snapshot),
-                    .ascending,
-                    null,
-                );
-                assert(it.next() == null);
+            for (manifest.levels) |_, level| {
+                manifest.assert_no_invisible_tables_at_level(@intCast(u8, level), snapshot);
             }
         }
 
+        fn assert_no_invisible_tables_at_level(
+            manifest: *const Manifest,
+            level: u8,
+            snapshot: u64,
+        ) void {
+            var it = manifest.levels[level].iterator(
+                .invisible,
+                @as(*const [1]u64, &snapshot),
```

Comment:
@ifreund is there a cleaner way to do this cast (as a follow up to this PR)?

The function signature expects `snapshots: []const u64`.

Reply:
I think it could be `&.{snapshot}`; I just wasn't paying enough attention when I moved that code.